Documentation/contributing/development/dev_setup.rst

```diff
@@ -211,9 +211,9 @@ If you want to connect to the Kubernetes cluster running inside the developer VM
 
 .. code-block:: shell-session
 
-    $ export KUBECONFIG=$KUBECONFIG:$GOPATH/src/github.com/cilium/cilium/vagrant.kubeconfig
+    $ export KUBECONFIG=$KUBECONFIG:$PATH_TO_CILIUM_REPO/vagrant.kubeconfig
 
-and add ``127.0.0.1 k8s1`` to your hosts file.
+where `$PATH_TO_CILIUM_REPO` is the path of your local clone of the Cilium git repository. Also add ``127.0.0.1 k8s1`` to your hosts file.
```

Comment:
Thanks!

```suggestion
where ``$PATH_TO_CILIUM_REPO`` is the path of your local clone of the Cilium git repository. Also add ``127.0.0.1 k8s1`` to your hosts file.
```

We need RST markup instead of Markdown here.

Reply:
Oh my bad, force of habbit. 😅 

Is it preferred that I fix and reabase my commit, or is adding a new one to update it fine?

Reply:
For Cilium, we usually fix and rebase (and since you ask, that would be my preference, as well :slightly_smiling_face:).

Reply:
Done, and rebased on c2b5e70535